services/github/github-deployments.tester.js

```diff
@@ -0,0 +1,18 @@
+'use strict'
+
+const Joi = require('@hapi/joi')
+const t = (module.exports = require('../tester').createServiceTester())
+
+t.create('Deployments')
+  .get('/badges/shields/shields-staging.json')
+  .expectBadge({
+    label: 'state',
+    message: Joi.string(),
```

Comment:
Can we define a more specific validation here.
For example, if the badge said `'environment not found'`, that would pass `Joi.string()`, but it shouldn't

Reply:
I do apologize, as you mentioned that before. For some reason .. I forgot again :) Fixed!